### PR TITLE
Subpage tabs

### PR DESCRIPTION
**Pull request by theilman7**
Created a pull request for the [Subpage Tabs component](https://oittfs.bcg.ad.bcgov.us/BAUCollection/BaltimoreCounty.Gov%20Website/_workitems/edit/15102) javascript. This adds the javascript file for a corresponding pull request in the [components repo](https://github.com/baltimorecounty/dotgov-components/pull/212).

### CODE REVIEW

**Comment by theilman7** [comment]
Updated to keep in line with PR in components repo. Will need to get with @martypowell tomorrow to see how to handle the error I get when attempting to resolve the conflicting files with git.

**Comment by martypowell** [comment]
> Updated to keep in line with PR in components repo. Will need to get with @martypowell tomorrow to see how to handle the error I get when attempting to resolve the conflicting files with git.

super easy but not apparent trick on this. Try rerunning the build, that should resolve any conflicts.

File: src/template/SubpageTabs.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", function() {
+  var menu, menuToggle, toggleButton;
+
+  menu = document.getElementById("dg_tabs");
```
Comment (martypowell) [inline src/template/SubpageTabs.js]: const menu = 
const menuToggle = 
const toggleButton =

File: src/components/SubpageTabs.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", function() {
```
Comment (martypowell) [inline src/components/SubpageTabs.js]: @theilman7 which one of these are we using? the seem to be doing the same thing?

File: src/components/SubpageTabs.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  var menu, menuToggle, toggleButton;
+
+  menu = document.getElementById("tabs");
+  menuToggle = document.getElementById("dg_menu-toggle");
+  toggleButton = document.getElementById("dg_menu-trigger-btn");
+
+  toggleButton.addEventListener("click", function(c) {
```
Comment (martypowell) [inline src/components/SubpageTabs.js]: if you are going to use shorthand typeically `e` is used for an event. We have been using the convention `clickEvent` for this type of variable.

This is esepcially useful when you do something like this, which we do in react.

```
const handleClick = clickEvent => {
  // do cool stuff here
}

toggleButton.addEventListener("click", handleClick);
```

Comment (theilman7) [reply src/components/SubpageTabs.js]: Updated and committed. Went with `clickEvent` as the variable. Let me know if this looks ok. @martypowell

File: src/components/SubpageTabs.js (hunk 2 above)
Comment (theilman7) [reply src/components/SubpageTabs.js]: Removed the unused variable, `menuToggle`. Remaining two are in use. Just pushed up for review @martypowell

File: src/template/SubpageTabs.js (hunk 1 above)
Comment (theilman7) [reply src/template/SubpageTabs.js]: Removed multi-var in favor of this style for variables. Update visible in most recent commit.
@martypowell

**Pull request merged**